projects/novo-elements/src/elements/data-table/cell-headers/data-table-header-cell.component.ts

```diff
@@ -283,6 +304,19 @@ export class NovoDataTableCellHeader<T> implements IDataTableSortFilter, OnInit,
     this.multiSelect = this.config.filterConfig && this.config.filterConfig.type ? this.config.filterConfig.type === 'multi-select' : false;
     if (this.multiSelect) {
       this.multiSelectedOptions = this.filter ? [...this.filter] : [];
+      if (typeof this.config.filterConfig.options[1] === 'string') {
+        (this.config.filterConfig.options as string[]).forEach(
```

Comment:
Could you use the map function here, i.e.
`this.multiSelectedOptionIsHidden = (this.config.filterConfig.options as string[]).map((option: string) => { option, hidden: false });`

Reply:
Certainly